tests/acceptance/features/apiSharingNg/mountOrUnmountShare.feature

```diff
@@ -86,3 +89,103 @@ Feature:  mount or unmount incoming share
       | resource      |
       | textfile0.txt |
       | folder        |
+
+
+  Scenario Outline: mounting shared resource by only one user in group shouldn't mount share for other user
+    Given user "Carol" has been created with default attributes and without skeleton files
+    And user "Alice" has disabled the auto-sync share
+    And user "Brian" has disabled the auto-sync share
+    And user "Carol" has uploaded file with content "hello world" to "/textfile0.txt"
+    And user "Carol" has created folder "FolderToShare"
+    And user "Carol" has sent the following share invitation:
+      | resource        | <resource> |
+      | space           | Personal   |
+      | sharee          | grp1       |
+      | shareType       | group      |
+      | permissionsRole | Viewer     |
+    When user "Alice" mounts share "<resource>" offered by "Carol" from "Personal" space using the Graph API
```

Comment:
And also i think this can be converted to given step. Good to have only one when step since we are checking result for mount share if i am not wrong?